Example/FancyShop/FancyShop/Modules/OrderSummary/Application Logic/Interactor/OrderSummaryInteractor.swift

```diff
@@ -136,7 +136,8 @@ class OrderSummaryInteractor:BaseInteractor, OrderSummaryInteractorInputProtocol
     }
     
     func initializeSdk() {
-        self.APIDataManager?.initializeSdk()
+//        self.APIDataManager?.initializeSdk(with: (self.presenter as? OrderSummaryPresenter)?.view as? UIViewController)
```

Comment:
Add comment this is an example

Reply:
comment added

Reply:
where is the comment I don't see it, Did you push your changes?